tools/offline_ota_image_builder/README.md

```diff
@@ -0,0 +1,173 @@
+# Offline OTA image builder
+
+`tools.offline_ota_image_builder` is a helper package for building offline OTA image from several input OTA images. Built image can be used as external cache source for otaproxy, or used by full offline OTA helper util to trigger fully offline OTA update.
+
+**NOTE**: full offline OTA helper util is not yet implemented.
+
+It provides the following features:
+
+1. build offline OTA image with several input OTA images,
+2. export built image as tar archive,
+3. prepare and export built image onto specific block device as external cache source dev.
+
+Please check documentation at [OTA cache mechanism design for otaproxy](https://tier4.atlassian.net/l/cp/yzmnPx9T) for more details.
```

Comment:
This link is not valid for public user. (It is OK if there is note that the link is for the internal developer)

Reply:
Emmm yes currently all documents are hosted on confluence. I will temporarily remove the link here.